src/files.js

```diff
@@ -37,14 +37,33 @@ const workspaceEnabled = loadState('text', 'workspace_enabled')
 registerFileCreate()
 
 document.addEventListener('DOMContentLoaded', () => {
+	function filesAppTriggersLoadViewerEvent() {
+		const [major, minor, maintenance] = OC.config.version.split('.').map(x => parseInt(x))
```

Comment:
just scrap this and make it 19 only, there is a stable18 branch, so master is okay to be 19 only?

Reply:
No idea, [master branch is currently compatible with 18 and 19](https://github.com/nextcloud/text/blob/d2a8ab5f3f9113ae69603efac5c24290d30a15f6/appinfo/info.xml#L30), so time to summon @juliushaertl

Reply:
Text is shipped, so master is 19 and 18 is stable18

Reply:
Yes, let's just do this on master which is 19 only.

Reply:
> master which is 19 only.

[Not really](https://github.com/nextcloud/text/blob/d2a8ab5f3f9113ae69603efac5c24290d30a15f6/appinfo/info.xml#L30) :-P Anyway I have removed the special handling for 18.

Reply:
Well yes, that should be fixed, but since we always release the text app together with the server I don't see issues with that :P